src/app/common/components/CertificateUpload.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { render } from 'react-dom';
+import {
+  Button,
+  InputGroup,
+  TextInput
+} from '@patternfly/react-core';
+
+interface ICertificateUploadProps {
+  isDisabled: boolean;
+  onBlur: any;
+  onChange: any;
+  onInput: any;
+  name: string;
+}
+
+interface ICertificateUploadState {
+  display: string;
+}
+
+class CertificateUpload extends React.Component<ICertificateUploadProps, ICertificateUploadState> {
+
+  private fileInput = React.createRef<HTMLInputElement>();
```

Comment:
We can use the react hook useRef for this

`  const fileInput = useRef(null);`